Foundation/NSSwiftRuntime.swift

```diff
@@ -147,10 +147,18 @@ internal func __CFSwiftGetBaseClass() -> UnsafeRawPointer {
     return unsafeBitCast(__NSCFType.self, to:UnsafeRawPointer.self)
 }
 
+#if os(Windows)
+@_silgen_name("__CFSocketInitializeWinSock")
```

Comment:
I doubt `@_silgen_name` will be allowed outside of stdlib. I think you need to use a `@_cdecl` instead.

Reply:
Can you use `@_cdecl` with a declaration and use it?  I don't want to put it into `__CFInitialize` since it will change the semantics of CoreFoundation outside of Foundation.

Reply:
Thinking about it, cant you just declare it in `ForSwiftFoundationOnly.h` ?
Alternatively when `__CFInitializeSwift` is called it is already gated on being inside of Foundation:
```
#if DEPLOYMENT_RUNTIME_SWIFT
        extern void __CFInitializeSwift(void);
        __CFInitializeSwift();
#endif
```
So could it just be added inside of this block?

Reply:
Yeah, I can declare it in `ForSwiftFoundationOnly.h`, that's perfectly fine by me.